packit_service/constants.py

```diff
@@ -38,7 +38,11 @@
 PG_COPR_BUILD_STATUS_FAILURE = "failure"
 PG_COPR_BUILD_STATUS_SUCCESS = "success"
 
-RETRY_LIMIT = 5
+RETRY_LIMIT = 2
+# retry in 3s, 6s
+RETRY_BACKOFF = 3
```

Comment:
Would it be hard to have it configurable?
- So we can have a different setup for prod/stage.
- Edits possible on the deployment level. => Quick change without code/image change and we can increase the numbers by steps.

Reply:
having it in the packit service config (or env vars) would be great

Reply:
no, that's a good point, will make it configurable